src/components/home/useful-links/useful-links.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import styles from './useful-links.module.css';
```

Comment:
My personal preference is to always have the style import be the very last import statement.